src/dag.c

```diff
@@ -325,3 +373,163 @@ int RAI_parseDAGPersistArgs(RedisModuleCtx *ctx, RedisModuleString **argv,
   }
   return argpos;
 }
+
+int RedisAI_DagRun_syntax_parser(RedisModuleCtx *ctx, RedisModuleString **argv,
```

Comment:
We should be more consistent with casing / underscores for function names throughout the module

Reply:
agree :) changing to `RedisAI_DagRunSyntaxParser`.